runtime/service/src/main/java/org/apache/polaris/service/admin/PolarisAdminService.java

```diff
@@ -672,6 +681,11 @@ private Map<String, SecretReference> extractSecretReferences(
                   AuthenticationParametersDpo.INLINE_BEARER_TOKEN_REFERENCE_KEY, secretReference);
               break;
             }
+          case SIGV4:
+            {
+              // SigV4 authentication is not secret-based
```

Comment:
Did you mean it's based on service identity? (which can be associated with a secret)

Reply:
Yes, we have OAuthAuthenticationParameters, which is based on user provided secrets.

Reply:
Revise the comment a bit!